testutil/encoding.go

```diff
@@ -14,6 +16,10 @@ func MustJSONMarshal(v any) []byte {
 	return b
 }
 
-func W(input string) []byte {
+type FmtStringer interface {
+	string | sdk.AccAddress | sdk.Int
+}
+
+func W[T FmtStringer](input T) []byte {
```

Comment:
If you want general wrapping, what about use `fmt.Stringer` interface instead of custom generic?

Reply:
I tried with `fmt.Stringer`at the first time.
But It didn't work for `string` which was a little bit surprising.
Someone said that Builtin types doesn't implement any interfaces(https://groups.google.com/g/golang-nuts/c/hLuHW-ApzH8/m/7tMedjvP6WwJ)

Reply:
Then what about this?

```suggestion
// W wraps input with double quotes if it is a string or fmt.Stringer.
func W(input any) []byte {
	switch input.(type) {
	case string, fmt.Stringer:
		return []byte(fmt.Sprintf("\"%s\"", input))
	default:
		panic("unsupported type")
	}
}
```

Reply:
But I think it's not good. because this change has error case and we can know it when run time. But in general type, we can know the error when compile time.

Reply:
Thanks for pointing out the pros and cons.
The function expects any argument that can be stringified. 
New comment seems to explain its purpose better.

Although it checks types at runtime, it doesn't need to add a new type to `string | sdk.AccAddress | sdk.Int` which is good point.

If you don't mind, I would keep it.